src/main/java/com/spotify/reaper/cassandra/JmxProxy.java

```diff
@@ -263,19 +270,19 @@ public int getPendingCompactions() {
     checkNotNull(cmProxy, "Looks like the proxy is not connected");
     try {
         ObjectName name = new ObjectName(COMP_OBJECT_NAME);
-        int pendingCount = (int) mbeanServer.getAttribute(name, "PendingTasks");
+        int pendingCount = (int) mbeanServer.getAttribute(name, VALUE_ATTRIBUTE);
         return pendingCount;
       } catch (IOException ignored) {
-        LOG.warn("Failed to connect to " + host + " using JMX", ignored);
+        LOG.warn(FAILED_TO_CONNECT_TO_USING_JMX, host, ignored);
       } catch (MalformedObjectNameException ignored) {
         LOG.error("Internal error, malformed name", ignored);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Internal error, malformed name" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)